fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -107,12 +82,15 @@ FixedLagSmoother::FixedLagSmoother(
 
   // Configure a timer to trigger optimizations
   optimize_timer_ = node_handle_.createTimer(
-    optimization_period_,
+    params_.optimization_period,
     &FixedLagSmoother::optimizerTimerCallback,
     this);
 
   // Advertise a service that resets the optimizer to its initial state
-  reset_service_server_ = private_node_handle_.advertiseService("reset", &FixedLagSmoother::resetServiceCallback, this);
+  reset_service_server_ = node_handle_.advertiseService(
```

Comment:
This switch from `private_node_handle_` to `node_handle_` is intentional. It is in the spirit of locusrobotics/fuse_rl#10